<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort</class>
 <widget class="QWidget" name="SerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="7,0">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_23">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>通信日志</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>810</width>
               <height>345</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QTextBrowser" name="recvEdit"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="recvNum">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>接收字节数量：0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearRecvBt">
              <property name="text">
               <string>清空内容</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>发送</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendBt">
              <property name="text">
               <string>发送测试数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="sendNum">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>发送字节数量：0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_25">
     <item>
      <widget class="QGroupBox" name="serialBox">
       <property name="title">
        <string>串口通信</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_24">
        <item>
         <widget class="QLabel" name="lblPortState">
          <property name="text">
           <string>串口待连接</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>端 口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="portNames"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="portSearchBt">
              <property name="text">
               <string>检测串口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="portOpenBt">
              <property name="text">
               <string>打开串口</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="wifiBox">
       <property name="title">
        <string>网络通信</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>IP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipInput">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>端口:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portInput">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblWifiState">
          <property name="text">
           <string>等待链接</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>协议类型:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="protocolComboBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="wifiConnectBt">
            <property name="text">
             <string>打开连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
